### DISCUSSION TITLE
报错问题

### DISCUSSION THREAD

**Discussion by sevenlemoon** [mikumifa/biliTickerBuy#203] [提问专区]
![image](https://github.com/mikumifa/biliTickerBuy/assets/174093696/be05ceea-822a-4249-a60f-9868bf80dbc9)
这是怎么回事呢，要怎么处理

**Comment by sevenlemoon** [comment]
@mikumifa

**Comment by mikumifa** [comment]
秋后算账了，以前违法请求过bw的账号全寄了
